src/Misc/layoutbin/en-US/strings.json

```diff
@@ -245,6 +246,13 @@
     "--username     basic authentication username",
     "--password     basic authentication password",
     "",
+    "",
+    "Arguments for enableautologon (Only on Windows):",
+    "-----------------------------------------------------------------------------",
+    "--restartnow          Used after autologon configuration is done. Recommended to restart the machine for the configured settings to take place.",
```

Comment:
we just want to keep it separate. It is different from service usage, even though screensaver is not disabled as UI testing can be done.

---
In reply to: [126191138](https://github.com/Microsoft/vsts-agent/pull/1063#discussion_r126191138) [](ancestors = 126191138)